content/chainguard/chainguard-enforce/chainguard-enforce-kubernetes/enforce-overview/index.md

```diff
@@ -0,0 +1,102 @@
+
+Violations can be sent as Cloud Events, so you can respond in the most appropriate way for your policies and applications. This might be to file a JIRA ticket, page the on-call, delete pods, or trigger a CI/CD pipeline to redeploy the application. Regardless of how you configure your response, the violation will be logged for future reference.
+
+> Note: For more information on how continuous verification works, check out our conceptual article on [Understanding Continuous Verification](../understanding-continuous-verification/).
+
+The following diagram outlines Chainguard Enforce's various components and how they work together. 
+
+![Diagram outlining Chainguard Enforce's architecture.](enforce-diagram.png)
+
+The Chainguard Enforce architecture contains a lightweight agent that can be installed in any Kubernetes cluster in a public or private cloud environment. The agent operates within the clusters to create, read, and apply policies. The Enforce Agent communicates with a centralized SaaS control plane where users can define, edit, and distribute policies across their fleet, as well as view information about their clusters, workloads, and policy results. The Enforce SaaS control plane also contains an Evidence Lake that allows organizations to store and retrieve information about their security posture over time. 
+
+Enforce can also be configured to send events to external systems like Slack, JIRA, or GitHub to notify users of policy violations, configuration changes, and more.  Finally, Enforce offers a command line interface, `chainctl`, for users to interact with clusters and policies from the command line.
+
+
+## Managing Chainguard Enforce with chainctl
+
+`chainctl` — short for "Chainguard Control" — is the command line interface tool for interacting with Chainguard Enforce. It communicates with the Enforce API, allowing you to manage policies, IAM groups, and cluster connections. Assuming you've been granted access to the product, you can [install `chainctl` for Chainguard Enforce](/chainguard/chainguard-enforce/how-to-install-chainctl/) on any machine you use to manage your Kubernetes clusters.
+
+`chainctl` uses the familiar `<context> <noun> <verb>` style of CLI interactions. For example, to list groups within the context of Chainguard IAM, you can run `chainctl iam groups list` to receive relevant output.
+
+For more information on how to use `chainctl`, please refer to our [`chainctl` documentation](/chainguard/chainguard-enforce/chainctl-docs/chainctl/).
+
+
+## The Chainguard Enforce Agent
```

Comment:
Do you think a page on the Enforce Agent that is a little more detailed is warranted?

Reply:
warranted, yes. But probably not a necessity right now